prettier/prettier.ts

```diff
@@ -1,7 +1,11 @@
 // Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+// @deno-types="./vendor/standalone.d.ts"
```

Comment:
Can you leave a comment explaining why you're giving types here? so we remember this issue in a few months?

Reply:
Done!